app/controllers/api/discussions_controller.rb

```diff
@@ -40,7 +46,8 @@ def collection_for_inbox(collection)
     collection.not_muted.unread.sorted_by_latest_activity
   end
 
-  def collection=(value)
-    @discussions = DiscussionWrapper.new_collection user: current_user, discussions: value
+  def discussion_event
```

Comment:
It's copied over from the old DiscussionReaderController, it's saying 'give me the event for the discussion with the given sequence id', if no sequence id is given, we get the discussion instead. It's used in the mark_as_read action